slimt/Annotation.hh

```diff
@@ -249,6 +272,22 @@ class AnnotatedText {
   std::string_view as_view(const Range &range) const {
     return std::string_view(text.data() + range.begin, range.size());
   }
+  Encoding encoding_{Encoding::Byte};
```

Comment:
Follow `encoding_ = Encoding::Byte`. This is nitpicking, but let's keep stuff consistent.

Reply:
https://github.com/jerinphilip/slimt/pull/38/commits/a41067721b637be7174284dd861cea46b79ae1eb